### PR TITLE
Replaced flake8 with pre-commit hooks and fixed code-style issues [PEP8]

### PR DESCRIPTION
**Pull request by amitsagtani97**
### Description
Added more code-quality checks for new pr's and 
fixed the existing code-style issues w.r.t PEP8 standard.

TODO 

- [x] Confirm this [FIXME](https://github.com/cloud-bulldozer/benchmark-wrapper/blob/master/snafu/fs_drift_wrapper/trigger_fs_drift.py#L115) statement and rsptime_date_str var.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      - id: check-merge-conflict
+      - id: end-of-file-fixer
+      - id: check-case-conflict
+      - id: detect-private-key
+      - id: check-ast
+
+  - repo: https://github.com/psf/black
+    rev: 19.10b0
+    hooks:
+      - id: black
```
Comment (rsevilla87) [inline .pre-commit-config.yaml:21]:
```suggestion
  - repo: https://github.com/psf/black
    rev: 19.10b0
    hooks:
      - id: black
        args: ['--line-length', '110'] 
```

black uses 88 characters as line length by default, for that reason is splitting lines shorter than 110
```  
-l, --line-length INTEGER       How many characters per line to allow.
                                  [default: 88]
```

Comment (amitsagtani97) [reply .pre-commit-config.yaml:21]: fixed :+1:

**Review by mohit-sheth — APPROVED** [review]
LGTM

**Review by rsevilla87 — APPROVED** [review]
LGTM!

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**